tests/test_expval.py

```diff
@@ -73,8 +73,10 @@ def test_paulix_expectation(self, theta, phi, qubit_device_3_wires, tol):
             rotations=[*O1.diagonalizing_gates(), *O2.diagonalizing_gates()],
         )
 
-        res = np.array([dev.expval(O1), dev.expval(O2)])
-        assert np.allclose(res, np.array([np.sin(theta) * np.sin(phi), np.sin(phi)]), tol)
+        res = np.array([dev.expval(O1), dev.expval(O2)], dtype=np.complex64)
```

Comment:
Is this co-opting a `complex128` test with `complex64` data? If so, I think we should keep the original and add a new one.